assets/css/dashboard/configure-screens-page.scss

```diff
@@ -0,0 +1,29 @@
+.configure-screens-page {
+  display: flex;
+  flex-direction: column;
+  color: $text-primary;
+  background: #0f1417;
+  min-height: 100vh;
+
+  .select-screen-type {
+    max-width: 1320px;
+
+    &__title {
+      white-space: nowrap;
+      width: fit-content;
+      font-weight: 700;
+      font-size: 28px;
+    }
+
+    .row {
+      justify-content: center;
+      gap: 24px;
+      margin-bottom: 32px;
+    }
+
+    .col {
+      width: fit-content;
+      max-width: 306px;
```

Comment:
This max-width on the column is not getting applied on the button-image itself. The effective max-width on the button is 282px.

I'm thinking the column / row approach is not the most flexible way of building this. Instead, maybe use flex, so that smaller viewports can shrink to have only 2 columns per row, [like this](https://www.figma.com/file/FKWRVrSdoMjZOLorhuykvc/Permanent-Configuration?type=design&node-id=1764-52754&mode=dev) (I know I never did my lunch chat on flex... 🙃 maybe time to do it?? lemme know if that'd still be helpful)

Reply:
The main reason for using the row/column approach is because it uses flex under the hood. With that said, I totally glossed over that image somehow. I thought that button sizes needed to shrink as the screen width decreased, but it actually just needs to reduce the number of columns. Bootstrap has what they call "Row Columns" to help with this. We can dictate the number of columns we want for each breakpoint. The `md` breakpoint is for widths `≥768px`. The `lg` is for `≥992px`. So if we say `lg={3}` and `md={2}`, we should see the behavior Mary designed. I will work on this and see what I come up with.